woocommerce/woocommerce
# Rebranding changes for core profiler and LYS

### Changes proposed in this Pull Request:

Closes #53373

Adds:

- `__experimentalFlags` in `wcSettings.admin` to populate with internally used flags to be consumed by JS client
- `isBranding2Enabled` utility function
- Core profiler and LYS logo changes
- DefaultFreeExtension changes

I chose to add `__experimentalFlags` instead of utilizing feature flag because the rebrand logic doesn't use feature flag.

### How to test the changes in this Pull Request:

#### Testing changes
1. Go to `WooCommerce > Settings > Advanced > WooCommerce.com`
2. Disable marketing suggestions
1. Install and activate [Code Snippets](https://wordpress.org/plugins/code-snippets/)
1. Add the following snippet and activate: `add_filter('woocommerce_use_new_branding', '__return_true');`
1. Go through core profiler, select `Alabama` as store country
3. In the extensions step, observe the header logo and WooCommerce tax & shipping logos are the **new logo**
4. Go to Homescreen and click on `Launch your store` task
5. Click on `Launch your store`
6. Observe the top logo is the **new logo**

#### Regression test
1. Deactivate the snippet from above instructions
1. Go through core profiler, select `Alabama` as store country
2. In the extensions step, observe the header logo and WooCommerce tax & shipping logos are the old logo
3. Go to Homescreen and click on `Launch your store` task
4. Click on `Launch your store`
5. Observe the top logo is the old logo

<img width="1414" alt="image" src="https://github.com/user-attachments/assets/57699944-0463-4d05-ad61-be3531c7ffb6">

<img width="1410" alt="image" src="https://github.com/user-attachments/assets/5b8f6556-15b7-4487-b6c5-eb7e4ed8fc20">



### Changelog entry












<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
